src/linesearch.jl

```diff
@@ -362,6 +362,8 @@ Convergence is not guaranteed in general.
 
 
 # Arguments
+- `inner_ls::LSM`: A fallback line search. Is only used if `safe==true`. (default `Backtracking`)
+- `safe::Bool`: Flag indicating whether the fallback line search should be used. (default `true`)  
```

Comment:
Maybe specify here what happens otherwise (the gamma from the last inner iteration is used)